api/v1/models/backend_address.go

```diff
@@ -29,6 +29,9 @@ type BackendAddress struct {
 
 	// Layer 4 port number
 	Port uint16 `json:"port,omitempty"`
+
+	// Backend weight
```

Comment:
Should this be updated to reflect the fact that only `0` and `1` values are currently supported?

Reply:
:+1: SG